docker-compose.yml

```diff
@@ -0,0 +1,60 @@
+version: "3.4"
+
+services:
+  server:
+    build:
+      context: ./backend
+      dockerfile: Dockerfile
+    volumes:
+      - ./backend/:/backend/
+    entrypoint: ./docker-start.sh
+    container_name: taia-x-backend
+    env_file:
+      - ./backend/.env
+    ports:
+      - 5678:5678
+      - 8000:8000
+    depends_on:
+      - db
+      - ipfs
+
+  frontend:
+    build:
+      context: ./frontend
+      dockerfile: Dockerfile
+    volumes:
+      - ~/.composer-docker/cache:/root/.composer/cache:delegated
+      - ./frontend/:/opt/taia-x/frontend:rw,delegated
+      - /opt/taia-x/frontend/node_modules
```

Comment:
@amrap030 I had to add also the path to the node_modules in to volumes for my container to access the dependencies installed in the image as explained in this [issue](https://github.com/vuejs/vue-cli/issues/3506#issuecomment-466915042)

Reply:
Ok looks good.